risc0/r0vm/src/lib.rs

```diff
@@ -35,6 +35,13 @@ struct Cli {
     #[arg(long, value_enum, default_value_t = HashFn::Sha256)]
     hashfn: HashFn,
 
+    /// When set to false, it is considered an error if the guest exists in an
+    /// unsuccessfully state, such as returning an exit code of `1` or
+    /// encountering a fault. When set to true, this is not considered an
+    /// error and proving will continue. Default is false.
```

Comment:
```suggestion
    /// error and proving will continue. Default is false.
    
```

Reply:
I assume this is because the paragraph clutters the help output? I address this in 2eb803d1a9af86266b034d2da1b4c5667b99ce5f